policyengine_us/variables/gov/irs/credits/earned_income/is_eitc_qualifying_child.py

```diff
@@ -13,9 +13,9 @@ def formula(person, period, parameters):
         disabled = person("is_permanently_and_totally_disabled", period)
         age = person("age", period)
         student = person("is_full_time_student", period)
-        max_ages = parameters(
+        p = parameters(
             period
-        ).gov.irs.credits.eitc.qualifying_child.max_age
-        max_age = where(student, max_ages.student, max_ages.non_student)
+        ).gov.irs.dependent.max_age
+        max_age = where(student, p.student, p.non_student)
         age_qualifies = age <= max_age
```

Comment:
if changing the values
```suggestion
        age_qualifies = age < max_age
```